etheno/genesis.py

```diff
@@ -106,10 +106,22 @@ def geth_to_parity(genesis):
 
     return ret
 
+DEFAULT_PRIVATE_KEYS = [0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d,
```

Comment:
How were these private keys chosen? Are they just random? If so, why not instead have `make_accounts` accept a seed (defaulting to a hard-coded value) and then generate an arbitrary number of accounts from a PRNG seeded with the given seed?

Reply:
Alternatively, you could just set the private keys incrementally from `0` through `num_accounts - 1`?

Reply:
I think these are the default ganache ones.

Reply:
I guess that's fine, then.